src/components/Footer/index.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { Fragment } from 'react';
+import styles from './styles.css';
```

Comment:
This is not the correct usage of `styled-component`. You should use a `javascript` file exporting your components stylized.

You can see the `Alert` component as an example: [styles.js](https://github.com/LucasCalazans/react-commerce/blob/master/src/components/Alert/styles.js)
```suggestion
import styles from './styles.js';
```